spec/lib/active_record/connection_adapters/abstract_adapter/connection_pool_spec.rb

```diff
@@ -1,206 +0,0 @@
-# -*- coding:binary -*-
-require 'spec_helper'
-
-RSpec.describe ActiveRecord::ConnectionAdapters::ConnectionPool do
```

Comment:
Just for context, what was the need for these tests previously? Does upgrading to rails 5.2 allow us to delete it now?

Reply:
History for this file shows it was created to validate a custom an ActiveRecord patch that was later removed.  This spec should have died long ago.